roles/edpm_iscsid/tasks/install.yml

```diff
@@ -43,16 +43,14 @@
         enabled: false
       when: stat_iscsid_socket.stat.exists
 
-    - name: Check if iscsi.service is enabled
-      ansible.builtin.command: systemctl is-enabled --quiet iscsi.service
-      failed_when: false
-      register: iscsi_service_enabled_result
+    - name: Gather services facts
+      ansible.builtin.service_facts:
```

Comment:
I'm OK with this, but bear in mind tripleo deployments struggled with performance and memory consumption issues, and one thing they did to mitigate the problem(s) was to avoid global fact gathering when the requirement was more specific. See commits like https://review.opendev.org/c/openstack/tripleo-ansible/+/775708.

In this instance, I don't care about ALL services, I just want to know about the iscsi service.

Reply:
It's a fair point and I totally understand it.

We'd like to avoid the use of "command" and "shell" modules when a native ansible module is available but obviously not at the price of deployment performance. 

I'd go with this "format" for now (since analog tasks are already been merged for all the other roles) but we'll keep this aspect monitored especially during scale tests. Let me know WDYT.

Reply:
I understand the tradeoffs, and am fine with using service_facts. We can revert to using the prior method if performance/scaling becomes a problem.